src/Contao/Hooks/LeafletLayerCallback.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Contao Leaflet MetaModels integration.
+ *
+ * @package    contao-leaflet-metamodels
+ * @author     Cliff Parnitzky <contao@cliff-parnitzky.de>
+ * @copyright  2015-2020 netzmacht David Molineus
+ * @license    LGPL 3.0-or-later https://github.com/netzmacht/contao-leaflet-metamodels/blob/master/LICENSE
+ * @filesource MetaModels\CoreBundle\Contao\Hooks\ModuleCallback
+ */
+
+namespace Netzmacht\Contao\Leaflet\MetaModels\Contao\Hooks;
```

Comment:
Please follow the existing naming convention. First of all, the class does not handle hooks but callbacks. Furthermore, both (hooks and dcs callback) are threated as listeners. A dca listener should live withing the namespace `Netzmacht\Contao\Leaflet\MetaModels\EventListener\Dca`.

Reply:
Renamed `Netzmacht\Contao\Leaflet\MetaModels\Contao\Hooks\LeafletLayerCallback` to `Netzmacht\Contao\Leaflet\MetaModels\EventListener\Dca\MetaModelsLeafletLayerDcaListener`